### ISSUE TITLE
Hide PDOM alerts from virtual cursor

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/balloons-and-static-electricity#223]
From #215

The aria-live alerts can be found with the virtual cursor.  We should try to hide them if possible, while still allowing the alerts to be announced by the AT.

One way to do this is to hide the aria-live content until text content changes on one of the aria-live elements.  Then immediately re-hide.

I believe we tried something like this before, and it caused issues.  But that was before we had the aria-live content in the PDOM prior to the document ```onload``` event firing (which caused many other issues), so this might work now.

**Comment by terracoda** [comment]
@jessegreenberg, would it be helpful to make the alerts visible during interaction and invisible during reading?

For the simple alerts (Two-balloon experiment, Remove/Add wall, Reset balloon(s), Reset All), for example, perhaps it is possible to make them visible upon activation of the button, and then after a short delay, hide them again, so if the user starts navigating with the virtual cursor after activating the button, the alert will be hidden again before they have time to find the alert content.

For the balloon alerts, it might be easier because presumably the user cannot "read" with the virtual cursor until the balloon is released.

**Comment by jessegreenberg** [comment]
> for example, perhaps it is possible to make them visible upon activation of the button, and then after a short delay, hide them again, so if the user starts navigating with the virtual cursor after activating the button, the alert will be hidden again before they have time to find the alert content.

I like that idea.  It has been a while, @terracoda can you please review this issue and let me know if this is still something that should be done?

**Comment by jessegreenberg** [comment]
@terracoda and I discussed.  We would like to proceed with this, but decision is dependent on a change in john-travoltage (https://github.com/phetsims/john-travoltage/issues/259), so adding on-hold label.

**Comment by jessegreenberg** [comment]
Alerts are now hidden after announced and cannot be found with the virtual cursor so this issue is stale and can be closed.
